### PR TITLE
Update Vulkan headers to 1.1

### PR DESCRIPTION
**Pull request by jason-simmons**

### CODE REVIEW

**Comment by jason-simmons** [comment]
@chinmaygarde

File: vulkan/vulkan_native_surface_android.cc [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 #include "flutter/vulkan/vulkan_native_surface_android.h"
 
+#include <android/native_window.h>
```
Comment (chinmaygarde) [inline vulkan/vulkan_native_surface_android.cc]: Did you mean to stage this? This doesn't seem related to the vulkan stuff.

Comment (jason-simmons) [reply vulkan/vulkan_native_surface_android.cc]: The old Vulkan headers were including `android/native_window.h`, but the new headers are not

**Comment by dnfield** [comment]
@jason-simmons since you reverted the Skia roll that required this, do we need to un-revert it as part of this PR?  Or is it fine either way?

**Comment by chinmaygarde** [comment]
Should be fine either way.

**Comment by jonahwilliams** [comment]
Trying to roll this into fuchsia and getting the following error while compiling for x64. Seems like one or the other should have been ifdef'd out?

```
../../third_party/flutter/vulkan/vulkan_interface.cc:82:10: error: duplicate case value: 'VK_ERROR_INVALID_EXTERNAL_HANDLE' and 'VK_ERROR_INVALID_EXTERNAL_HANDLE_KHR' both equal '-1000072003'
    case VK_ERROR_INVALID_EXTERNAL_HANDLE_KHR:
         ^
../../third_party/flutter/vulkan/vulkan_interface.cc:65:10: note: previous case defined here
    case VK_ERROR_INVALID_EXTERNAL_HANDLE:
         ^
../../third_party/flutter/vulkan/vulkan_interface.cc:88:10: error: duplicate case value: 'VK_ERROR_OUT_OF_POOL_MEMORY' and 'VK_ERROR_OUT_OF_POOL_MEMORY_KHR' both equal '-1000069000'
    case VK_ERROR_OUT_OF_POOL_MEMORY_KHR:
         ^
../../third_party/flutter/vulkan/vulkan_interface.cc:67:10: note: previous case defined here
    case VK_ERROR_OUT_OF_POOL_MEMORY:
         ^
```

**Comment by jonahwilliams** [comment]
cc @chinmaygarde / @jason-simmons

**Comment by jason-simmons** [comment]
@egdaniel
